### DISCUSSION TITLE
no eol found yet, partial frame

### DISCUSSION THREAD

**Discussion by tomtomtom2** [wmbusmeters/wmbusmeters#1315] [General]
Hi!

I have the following problem.

`(serial) received binary "6238453434463035433738333630333230304430373541463137413134303038303035"
(cul) checkCULFrame "b8E44F05C783603200D075AF17A14008005"
(cul) no eol found yet, partial frame
(serial) received binary "4644383638413338344230433842463542453044303741433132454332354239363030463145413033443630333946314146324637454433303435"
(cul) checkCULFrame "b8E44F05C783603200D075AF17A14008005FD868A384B0C8BF5BE0D07AC12EC25B9600F1EA03D6039F1AF2F7ED3045"
(cul) no eol found yet, partial frame
(serial) received binary "454441423130324434303137423642424634424331443245314546443741344341333944343038394145383833464137334538424232314534344433423435433239334232434636384537413035353435344630323142433634413239443845303341384436464446413539423738423835364436453137363741"
(cul) checkCULFrame "b8E44F05C783603200D075AF17A14008005FD868A384B0C8BF5BE0D07AC12EC25B9600F1EA03D6039F1AF2F7ED3045EDAB102D4017B6BBF4BC1D2E1EFD7A4CA39D4089AE883FA73E8BB21E44D3B45C293B2CF68E7A055454F021BC64A29D8E03A8D6FDFA59B78B856D6E1767A"
(cul) no eol found yet, partial frame
(serial) received binary "3931303744383130423644423843353832374331463745363834383043303239334246444239414136384234453544444231393138413430433542323939313836353746323942443438344138304644320D0A"
(cul) checkCULFrame "b8E44F05C783603200D075AF17A14008005FD868A384B0C8BF5BE0D07AC12EC25B9600F1EA03D6039F1AF2F7ED3045EDAB102D4017B6BBF4BC1D2E1EFD7A4CA39D4089AE883FA73E8BB21E44D3B45C293B2CF68E7A055454F021BC64A29D8E03A8D6FDFA59B78B856D6E1767A9107D810B6DB8C5827C1F7E68480C0293BFDB9AA68B4E5DDB1918A40C5B29918657F29BD484A80FD2<0D><0A>"
(cul) checkCULFrame RSSI_RAW=-46
(cul) checkCULFrame LQI=7
[2024-07-03_13:10:56] (cul) Warning! Your cul firmware has a bug that prevents longer telegrams from being received.!
[2024-07-03_13:10:56] (cul) Please read: https://github.com/wmbusmeters/wmbusmeters/issues/390
[2024-07-03_13:10:56] (cul)         and: https://wmbusmeters.github.io/wmbusmeters-wiki/nanoCUL.html
(cul) bad hex "8E44F05C783603200D075AF17A14008005FD868A384B0C8BF5BE0D07AC12EC25B9600F1EA03D6039F1AF2F7ED3045EDAB102D4017B6BBF4BC1D2E1EFD7A4CA39D4089AE883FA73E8BB21E44D3B45C293B2CF68E7A055454F021BC64A29D8E03A8D6FDFA59B78B856D6E1767A9107D810B6DB8C5827C1F7E68480C0293BFDB9AA68B4E5DDB1918A40C5B29918657F29BD484A8"
[2024-07-03_13:10:56] (cul) warning: the hex string is not proper! Ignoring telegram!
(cul) error in received message.
(serial) received binary "623845343446303543373933363033323030443037394237393741373330303830303534353742383738"
(cul) checkCULFrame "b8E44F05C793603200D079B797A73008005457B878"
(cul) no eol found yet, partial frame
(serial) received binary "43413843314339464232333242343745463731373831464630383837443636333736383342433343334442414338374633393841324442463542303534343139413841333135373132443639324135463137324542443834423445453144324436393442303744364632393039463042313736384232353036"
(cul) checkCULFrame "b8E44F05C793603200D079B797A73008005457B878CA8C1C9FB232B47EF71781FF0887D6637683BC3C3DBAC87F398A2DBF5B054419A8A315712D692A5F172EBD84B4EE1D2D694B07D6F2909F0B1768B2506"
(cul) no eol found yet, partial frame
(serial) received binary "463239313636424536413646453233443137353241394341343135424244453433353644433344323341433338384344374241353330334635463545"
(cul) checkCULFrame "b8E44F05C793603200D079B797A73008005457B878CA8C1C9FB232B47EF71781FF0887D6637683BC3C3DBAC87F398A2DBF5B054419A8A315712D692A5F172EBD84B4EE1D2D694B07D6F2909F0B1768B2506F29166BE6A6FE23D1752A9CA415BBDE4356DC3D23AC388CD7BA5303F5F5E"
(cul) no eol found yet, partial frame
(serial) received binary "3041323031313841423643394437374231464133333943423536373045453535433345323639463735413539314435393444384446443435423536373541423530393734383943423637320D0A"
(cul) checkCULFrame "b8E44F05C793603200D079B797A73008005457B878CA8C1C9FB232B47EF71781FF0887D6637683BC3C3DBAC87F398A2DBF5B054419A8A315712D692A5F172EBD84B4EE1D2D694B07D6F2909F0B1768B2506F29166BE6A6FE23D1752A9CA415BBDE4356DC3D23AC388CD7BA5303F5F5E0A20118AB6C9D77B1FA339CB5670EE55C3E269F75A591D594D8DFD45B5675AB5097489CB672<0D><0A>"
(cul) checkCULFrame RSSI_RAW=114
(cul) checkCULFrame LQI=91
[2024-07-03_13:11:52] (cul) Warning! Your cul firmware has a bug that prevents longer telegrams from being received.!
[2024-07-03_13:11:52] (cul) Please read: https://github.com/wmbusmeters/wmbusmeters/issues/390
[2024-07-03_13:11:52] (cul)         and: https://wmbusmeters.github.io/wmbusmeters-wiki/nanoCUL.html
(cul) bad hex "8E44F05C793603200D079B797A73008005457B878CA8C1C9FB232B47EF71781FF0887D6637683BC3C3DBAC87F398A2DBF5B054419A8A315712D692A5F172EBD84B4EE1D2D694B07D6F2909F0B1768B2506F29166BE6A6FE23D1752A9CA415BBDE4356DC3D23AC388CD7BA5303F5F5E0A20118AB6C9D77B1FA339CB5670EE55C3E269F75A591D594D8DFD45B5675AB5097489C"
[2024-07-03_13:11:52] (cul) warning: the hex string is not proper! Ignoring telegram!
(cul) error in received message.
(serial) received binary "62384534344630354337383336303332303044303735414631374131353030383030353044364637364246393136313837383437353131463145"
(cul) checkCULFrame "b8E44F05C783603200D075AF17A150080050D6F76BF916187847511F1E"
(cul) no eol found yet, partial frame
(serial) received binary "4237323744373533393733393436324138443543424136333344324130333445353545363746373433304239323343313335464341413639413045384238414431393435303541364444443435453046443246393735463837453031414132434231383442354544433644313542433241333230353532463645453338363330313744454135424138454239373635454635363932314443323532354244343731374543463742414138324441393037383238453845394544"
(cul) checkCULFrame "b8E44F05C783603200D075AF17A150080050D6F76BF916187847511F1EB727D7539739462A8D5CBA633D2A034E55E67F7430B923C135FCAA69A0E8B8AD194505A6DDD45E0FD2F975F87E01AA2CB184B5EDC6D15BC2A320552F6EE3863017DEA5BA8EB9765EF56921DC2525BD4717ECF7BAA82DA907828E8E9ED"
(cul) no eol found yet, partial frame
(serial) received binary "434335333838463142303441413037374131414236383742343944384441313131453535374234313041433937414541413336393134410D0A"
(cul) checkCULFrame "b8E44F05C783603200D075AF17A150080050D6F76BF916187847511F1EB727D7539739462A8D5CBA633D2A034E55E67F7430B923C135FCAA69A0E8B8AD194505A6DDD45E0FD2F975F87E01AA2CB184B5EDC6D15BC2A320552F6EE3863017DEA5BA8EB9765EF56921DC2525BD4717ECF7BAA82DA907828E8E9EDCC5388F1B04AA077A1AB687B49D8DA111E557B410AC97AEAA36914A<0D><0A>"
(cul) checkCULFrame RSSI_RAW=74
(cul) checkCULFrame LQI=72
[2024-07-03_13:12:12] (cul) Warning! Your cul firmware has a bug that prevents longer telegrams from being received.!
[2024-07-03_13:12:12] (cul) Please read: https://github.com/wmbusmeters/wmbusmeters/issues/390
[2024-07-03_13:12:12] (cul)         and: https://wmbusmeters.github.io/wmbusmeters-wiki/nanoCUL.html
(cul) bad hex "8E44F05C783603200D075AF17A150080050D6F76BF916187847511F1EB727D7539739462A8D5CBA633D2A034E55E67F7430B923C135FCAA69A0E8B8AD194505A6DDD45E0FD2F975F87E01AA2CB184B5EDC6D15BC2A320552F6EE3863017DEA5BA8EB9765EF56921DC2525BD4717ECF7BAA82DA907828E8E9EDCC5388F1B04AA077A1AB687B49D8DA111E557B410AC97AEAA36"
[2024-07-03_13:12:12] (cul) warning: the hex string is not proper! Ignoring telegram!
(cul) error in received message.`

I already flashed the new firmware. Can somebody tell me what to do?

**Comment by weetmuts** [comment]
The nanoCUL can never handle full-length telegrams. The "fixed" or should we say "improved" firmware can only handle slightly longer telegrams. Sadly. Buy a better dongle.

**Comment by weetmuts** [comment]
I have updated the wiki https://wmbusmeters.github.io/wmbusmeters-wiki/nanoCUL.html with a big warning about this!
